source/docs/v3/tenancy-bootstrappers.blade.md

```diff
@@ -29,31 +29,77 @@ php artisan cache:clear --tag=tenant_123
 Note that you must use a cache store that supports tagging, e.g. Redis.
 
 ## Filesystem tenancy bootstrapper {#filesystem-tenancy-boostrapper}
+The filesystem bootstrapper makes your app's `storage_path()` and `asset()` helper and the `Storage` facade tenant-aware by modifying the paths they retrieve.
 
-This bootstrapper does the following things:
+> Note: If you want to bootstrap filesystem tenancy differently (e.g. provision an S3 bucket for each tenant), you can absolutely do that. Take a look at the package's bootstrappers to get an idea of how to write one yourself, and feel free to implement it any way you want.
+
+### Storage path helper
+
+The bootstrapper suffixes the path retrieved by `storage_path()` to make the helper tenant-aware.
+
+- The suffix is built by appending the tenant key to your `suffix_base`. The `suffix_base` is `tenant` by default, but feel free to change it in the `tenancy.filesystem` config. For example, the suffix will be `tenant42` if the tenant's key is `42` and the `suffix_base` is `tenant`.
+- After the suffixing, `storage_path()` helper returns `"/$path_to_your_application/storage/$suffix/"`
+
+Since `storage_path()` will be suffixed, your folder structure will look like this:
+
+![The folder structure](https://i.imgur.com/GAXQOnN.png)
```

Comment:
Could you move the image to our repo (https://github.com/stancl/tenancy-docs/tree/master/source/assets/images) and link to it like this https://github.com/stancl/tenancy-docs/blob/master/source/docs/v3/early-identification.blade.md?plain=1#L71

It will not display in this PR until it's merged, but after that it will work and it will be a reliable part of our repo.

Reply:
The same imgur link was used for v1 and v2 docs too, so I changed the links there too.